### PR TITLE
Adds a drone arena prefab

### PR DESCRIPTION
**Pull request by unfunnyperson**
[FEATURE]
## About the PR 
Adds a rare prefab. Its a semi-large arena with a powerful drone in the middle. Killing the drone drops a id card which you can use to get into the building that has a neat gun inside of it.

## Why's this needed? 
We could always use more prefabs.

## Changelog

```changelog
(u)UnfunnyPerson
(+)Added a space drone arena prefab
```

### CODE REVIEW

**Comment by unfunnyperson** [comment]
image

**Comment by Tarmunora** [comment]
personally, can't say I'm a fan of well, any of this.

**Comment by Bram-KEP** [comment]
It looks cool, but maybe add some things to fight the drone with?
Also dropping agent card is not Great, perhaps the door could use a custom one that is dropped?

**Comment by Bram-KEP** [comment]
Seems ok now
Also the gun will have to be added to the wiki.

File: code/z_adventurezones/dronearena.dm [hunk 1]
```diff
@@ -82,6 +82,14 @@
 			elecflash(src.loc,1,1)
 			playsound(src.loc, "sound/machines/glitch1.ogg", 50, 1)
 
+/obj/machinery/door/airlock/pyro/syndicate/dronearena
+	req_access_txt = "85"
+
+/obj/item/card/id/dronearena
+	name = "old id card"
+	desc = "A dusty old id card. Is this id even working right?!"
```
Comment (Zonespace27) [inline code/z_adventurezones/dronearena.dm:90]: Small Nitpick: capitalizing ID would be good for consistency's sake.

**Comment by Bram-KEP** [comment]
Also merge conflict, you may want to fix that. It seems pretty good now although if it's supposed to be a syndicate base make the floors red along with other stuff perhaps, so it looks more in theme.

**Comment by Tarmunora** [comment]
Not interested in the idea.

**Comment by Kyle2143** [comment]
This PR is a bit lacking in description. It should contain everything new added in so we don't have to read all the code to get the gist. The inclusion of this new weapon is arguably even more big than the prefab itself.

I like the concept of combat contest based prefabs to get some reward. But I think that this specific reward is too strong and the contest too easy, especially for the amount of ammunition you get upon winning. Just build a halfway decent pod and grab a decent melee weapon+meds and you're fine. If you have a partner then it's trivial.

As for the drone enemy itself, we already have a lot of enemy drones in space, I'd rather see something like having to fight armed humans or even just randomized dangerous critters.

**Pull request closed without merging**